webauthntest.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
```

Comment:
do we need this file? if yes, maybe add a comment that tells us what it's supposed to do?